### DISCUSSION TITLE
vulnerability in parse-path dependency

### DISCUSSION THREAD

**Discussion by eukevin** [nuxt/nuxt#17316] [Questions] [answered]
```
macOS 12.4
npm 8.12.1
node v18.3.0
```

Running `npm audit` against `3.0.0-rc.4` results in the following:
```
# npm audit report

parse-path  <5.0.0
Severity: moderate
Authorization Bypass in parse-path - https://github.com/advisories/GHSA-3j8f-xvm3-ffx4
fix available via `npm audit fix --force`
Will install nuxt@2.12.2, which is a breaking change
node_modules/parse-path
  parse-url  3.0.0 - 6.0.2
  Depends on vulnerable versions of parse-path
  node_modules/parse-url
    git-up  2.1.0 - 5.0.0
    Depends on vulnerable versions of parse-url
    node_modules/git-up
      git-url-parse  11.0.0 - 11.6.0
      Depends on vulnerable versions of git-up
      node_modules/git-url-parse
        @nuxt/telemetry  *
        Depends on vulnerable versions of git-url-parse
        node_modules/@nuxt/telemetry
          nuxt  >=2.13.0
          Depends on vulnerable versions of @nuxt/telemetry
          node_modules/nuxt

6 moderate severity vulnerabilities
```

Is this something that needs to be/will be addressed, or is this something we can safely ignore?

**Comment by danielroe** [comment, marked as answer]
As the security issue in `parse-url` was fixed in a patch release, you only have to run `yarn upgrade` (or npm/pnpm equivalent) in your project to benefit from the update. Likewise, any new Nuxt projects will be using the newer version automatically.

Moreover, Nuxt is not affected by this vulnerability. The URL that is parsed is from the developer's machine, and even if any remote URLs are wrongly parsed, it will just change the internal project hash.
